src/components/TopicLoader/TopicLoader.js

```diff
@@ -70,6 +70,10 @@ class TopicLoader extends Component {
   onClick(features, layer, event) {
     const { dispatchSetClickedFeatureInfo } = this.props;
 
+    if (!layer.getVisible() || !features.length) {
```

Comment:
Please remove `state.clickedFeatureInfo` is an empty array, it means no features are clicked.
Moreover, invisible can't be clicked which is why checking the visibility of the layer is not necessary here.

Reply:
@gevos do you forgot to make changes for this comment ?

Reply:
@gevos please update this.